src/rush.ts

```diff
@@ -1,40 +1,75 @@
+import stripJsonComments from "strip-json-comments";
 import { npmSearchGenerator } from "./npm";
 
+interface IRushConfigurationProjectJson {
+  packageName: string;
+}
+interface IRushConfigurationJson {
+  projects: IRushConfigurationProjectJson[];
+}
+const projectGenerator: Fig.Generator = {
+  script:
+    "until [[ -f rush.json ]] || [[ $PWD = '/' ]]; do cd ..; done; cat rush.json",
+  postProcess: function (out: string) {
+    const suggestions = [];
+
+    try {
+      if (out.trim() == "") {
+        return suggestions;
+      }
+
+      const rushConfigurationJson: IRushConfigurationJson = JSON.parse(
+        stripJsonComments(out)
+      );
```

Comment:
`out.trim()` does not mutate the original string:
```suggestion
      if (!out) {
        return suggestions;
      }

      const rushConfigurationJson: IRushConfigurationJson = JSON.parse(
        stripJsonComments(out.trim())
      );
```

Reply:
fixed! thanks for review